pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentRestletResource.java

```diff
@@ -591,7 +592,8 @@ private void deleteSegmentsInternal(String tableNameWithType, List<String> segme
   @ApiOperation(value = "Get the server metadata for all table segments", notes = "Get the server metadata for all table segments")
   public String getServerMetadata(
       @ApiParam(value = "Name of the table", required = true) @PathParam("tableName") String tableName,
-      @ApiParam(value = "OFFLINE|REALTIME") @QueryParam("type") String tableTypeStr) {
+      @ApiParam(value = "OFFLINE|REALTIME") @QueryParam("type") String tableTypeStr,
```

Comment:
(nit) please add javadocs also indicating how the column list is used and what is the behavior when user doesn't specify it (leaves it to default)

Reply:
javadocs is not needed here since swagger API docs will be auto generated.